Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'guard-shell'
+gem 'rubocop'
```

Comment:
Rubocop is by no means _necessary_ in this case, but let's just use it since it's good practice in the Ruby world (where I spend most of my work time anyway). If/when we add Travis CI, we could as well run the linting there.

(Would be cool to have something like Rubocop for Rust also; I don't think it exists _yet_. I guess I will have to write it. :smile:)